global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sdk": {
+      "version": "6.0.413",
```

Comment:
🎨 After some good discussion on the topic of runtime support being included in SDK updates (https://bitwarden.slack.com/archives/C04LE162PHA/p1699296762613869) this can be `6.0.100`.

Reply:
I was having trouble building out project in WSL because of this. thanks for the link.

Reply:
CC @justindbaur 😜